### PR TITLE
fix for watcher issue

### PR DESCRIPTION
**Pull request by dtregonning**

### CODE REVIEW

**Comment by dtregonning** [comment]
Screen Shot 2019-03-29 at 11 08 30 AM

We've had 3-4 days of success with Watcher Objects

**Comment by dtregonning** [comment]
Credit to @peter-moffet

File: lib/fluent/plugin/in_kubernetes_objects.rb [hunk 1]
```diff
@@ -72,6 +72,9 @@ class KubernetesObjectsInput < Fluent::Plugin::Input
 
       desc 'A selector to restrict the list of returned objects by fields.'
       config_param :field_selector, :string, default: nil
+
+      desc 'The interval at which the objects will be watched.'
+      config_param :interval, :time, default: 15 * 60
```
Comment (cben) [inline lib/fluent/plugin/in_kubernetes_objects.rb]: Hi, I want to learn from your code, and your experience, towards handling watch resumption in kubeclient itself (https://github.com/abonas/kubeclient/issues/273).

I'm curious why you chose to wait 15 minutes here before attempting to re-watch.
My impression is recent k8s versions using etcd3 only keep old resourceVersion around for ~5min, so would expect this to fail with "410 Gone" errors.  Have you seen those?

[I'm aware this code has been superceded by #34 which now immediately loops to watch again.  Just curious about the story :-]

**Comment by fradee** [comment]
https://github.com/splunk/fluent-plugin-kubernetes-objects/issues/44

**Pull request merged**